src/internal/observable/iif.ts

```diff
@@ -90,7 +94,7 @@ import { SubscribableOrPromise } from '../types';
  * @static true
  * @name iif
  * @owner Observable
- */
+*/
 export function iif<T, F>(
```

Comment:
seems like you could just put the default generics down here, and you wouldn't need the line up top.